client/state/data-layer/wpcom/activity-log/rewind/restore-status/index.js

```diff
@@ -39,6 +39,10 @@ const fetchProgress = action => {
 	const lastUpdate = recentRequests.get( key ) || -Infinity;
 	const now = Date.now();
 
+	if ( ! restoreId ) {
```

Comment:
Wouldn't we want to this check even higher up? So that we don't end up doing https://github.com/Automattic/wp-calypso/blob/12c8f9fd7e02e442263eaf1553f6ebf7adca41ed/client/state/data-layer/wpcom/activity-log/rewind/restore-status/index.js#L39 ?

Reply:
that shouldn't matter, because the `get` in this case is not a network request.